overrides/cf.d.ts

```diff
@@ -306,6 +306,7 @@ interface IncomingRequestCfProperties {
    */
   country: string;
   httpProtocol: string;
+  isEUCountry: boolean;
```

Comment:
This type doesn't really seem consistent with what the docs describe. I feel like it should be something like this:
```suggestion
  isEUCountry?: string;
```

Reply:
sure thing, let me double check the actual object as first though I feel this was triggered because I had a boolean (not a string)

Reply:
you're right, this is coming back as `"isEUCountry":"1"` or `"isEUCountry":null` :sigh: good catch!